src/test/java/com/lberkholtz/advancedava/week3/Main/StockQuoteApplicationTest.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * @author Laura Berkholtz
+ */
+
+package com.lberkholtz.advancedava.week3.Main;
+
+import org.junit.Test;
+
+import java.text.ParseException;
+
+/**
+ * Tests the StockQuoteApplication class
+ */
+
+public class StockQuoteApplicationTest {
+    String[] args = {"APPL", "2018-01-01", "2018-01-10"};
+
+    @Test
+    public void main()throws ParseException {
+        StockQuoteApplication.main(args);
```

Comment:
use a better method name for the test. 
also, test for excepted exceptions like this:
  @Test(expected = IllegalArgumentException.class)
    public void testNegative() {
        App app = new App(null);
        assertNull("Verify the message is set correctly", app.getMessage());

Reply:
Not sure about this. I'll have to talk to you about it in the 1 on 1.